scripts/update-bcd.ts

```diff
@@ -814,11 +815,57 @@ const skipCurrentBeforeSupport = skip("currentBeforeSupport", ({
   }
 });
 
+/**
+ *  Iterates through an array of default support statements to detect support for a specific browser version.
+ * @param version - A version string.
+ * @param hasSupport - An boolean indicating if a test result for the version shows support.
+ * @param statements - An array of default statements.
+ * @returns A boolean indicating whether default statements indicate support for the version.
+ */
+const isSupported = (
+  version: string,
+  hasSupport: boolean,
+  statements: SimpleSupportStatement[],
+) => {
+  for (const {version_added, version_removed} of statements) {
+    if (version_added === "preview") {
+      return false;
+    }
+
+    // In the case of general boolean statements, only show support if the version from the test result does not show specific support, otherwise we should ignore generic boolean support statements in favor of specific version support info from a test result
```

Comment:
Yeah, I agree this makes sense. It's a bit unfortunate that `hasSupport` is passed in, but it makes sense here. Maybe it can be refactored somehow, but I'm not sure how right now, and this works!